### PR TITLE
Rem:remove meshrouter and v3router 

### PR DESCRIPTION
**Pull request by pherzheyu**
**What this PR does**: 
remove meshrouter and v3router

**Which issue(s) this PR fixes**: 
Fixes #

**You should pay attention to items below to ensure your pr passes our ci test**
We do not merge pr with ci tests failed

- [x] All ut passed (run 'go test ./...' in project root)
- [x] After go-fmt ed , run 'go fmt project' using goland.
- [x] Golangci-lint passed, run 'sudo golangci-lint run' in project root.
- [x] After import formatted, (using [imports-formatter](https://github.com/dubbogo/tools#5-how-to-get-imports-formatter) to run 'imports-formatter .' in project root, to format your import blocks, mentioned in [CONTRIBUTING.md](https://github.com/apache/dubbo-go/blob/master/CONTRIBUTING.md) above) 
- [x] Your new-created file needs to have [apache license](https://raw.githubusercontent.com/dubbogo/resources/master/tools/license/license.txt) at the top, like other existed file does.
- [x] All integration test passed. You can run integration test locally (with docker env). Clone our [dubbo-go-samples](https://github.com/apache/dubbo-go-samples) project and replace the go.mod to your dubbo-go, and run 'sudo sh start_integration_test.sh' at root of samples project root. (M1 Slice is not Support)

### CODE REVIEW

File: registry/directory/directory_test.go [hunk 1]
```diff
@@ -206,25 +203,4 @@ func TestToGroupInvokers(t *testing.T) {
 		time.Sleep(1e9)
 		assert.True(t, len(registryDirectory.toGroupInvokers()) == 2)
 	})
-
-	t.Run("DifferentGroup", func(t *testing.T) {
```
Comment (LaurenceLiZhixin) [inline registry/directory/directory_test.go:210]: 这个为什么删了？

Comment (pherzheyu) [reply registry/directory/directory_test.go:210]: 看起来像是测试v3router的

Comment (sanxun0325) [reply registry/directory/directory_test.go:210]: 这个不需要删，主要是为了测试不同group转换Invoker的情况

**Review by sanxun0325 — COMMENTED** [review]
common/constant#key.go 下v3router相关的常量可以删

**Review by AlexStocks — APPROVED** [review]

File: registry/directory/directory_test.go (hunk 1 above)
Comment (pherzheyu) [reply registry/directory/directory_test.go:210]: 已修改

**Comment by pherzheyu** [comment]
> common/constant#key.go 下v3router相关的常量可以删

已删除

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by sanxun0325 — APPROVED** [review]
LGTM

**Pull request merged**